frontend/components/media/MediaMap.vue

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Going through this now, @rabelmervin. Firstly we don't need the extra spaces here :)